src/devices/imou.ts

```diff
@@ -73,4 +73,11 @@ export const definitions: DefinitionWithExtend[] = [
         description: "Wireless switch",
         extend: [m.battery(), imouAlarmButton()],
     },
+    {
+        zigbeeModel: ['ZD2-EN'],
+        model: 'ZD2-EN',
+        vendor: 'MultIR',
+        description: 'Automatically generated definition',
+        extend: [m.battery(), m.iasZoneAlarm({"zoneType":"generic","zoneAttributes":["alarm_1","alarm_2","tamper","battery_low"]})],
```

Comment:
Can you use something else as `generic` here? Possible options can be found here: https://github.com/Koenkk/zigbee-herdsman-converters/blob/802c2d97669f56ad94fa9592b688848efea71d9c/src/lib/modernExtend.ts#L1605